data/events/2022-eindhoven.yml

```
@@ -0,0 +1,114 @@
+name: "2022-eindhoven" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2022" # The year of the event. Make sure it is in quotes.
+city: "Eindhoven" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsdaysehv" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Devopsdays is coming to Eindhoven!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2022-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2022-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2022-10-19T00:00:00+02:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2022-10-21T00:00:00+02:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: 2021-07-07T00:00:00+02:00 # start accepting talk proposals.
+cfp_date_end: 2021-09-21T00:00:00+02:00 # close your call for proposals.
+cfp_date_announce: # inform proposers of status
+
+cfp_link: "https://sessionize.com/devopsdays-eindhoven-2022/" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+
+registration_date_start: # start accepting registration. Leave blank if registration is not open yet. This will make the "Register" button appear on your "Welcome" page.
+registration_date_end: # close registration. Leave blank if registration is not open yet. If you set "registration_date_start" you need a value here.
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "Eindhoven" # Defaults to city, but you can make it the venue name.
+#
+location_address: "High Tech Campus 1b, 5656 AE Eindhoven, The Netherlands" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+masthead_background: "eindhoven-bg.jpeg"
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  # - name: propose
+  - name: location
```

As you haven't filled out `location.md`, either do that _or_ leave this commented out for now.

```suggestion
  # - name: location
```

---

This page is now updated